hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -966,6 +1014,9 @@ def setup_new_composite_areas(
                 self.crs,
             )
 
+        # Adding elevation data into the new objects
+        self.setup_ground_elevation(ground_elevation,new_objects,_new_exposure_geoms)
```

Comment:
Very tiny thing but we can do formatting with Ruff by running `ruff .` in the main folder of the repository.
```suggestion
        self.setup_ground_elevation(ground_elevation, new_objects, _new_exposure_geoms)
```